### DISCUSSION TITLE
Webinterface with PHP 7.0.x isn't working

### DISCUSSION THREAD

**Discussion by bpgs** [datenstrom/community#172] [Report a bug] [answered]
Recently I've installed XAMPP 7.0.4 (with PHP 7.0.4) on Windows 7 and now I can't login / edit the Yellow content pages. If I try to login nothing happens. In the error Log I've found this warning
`setcookie() expects parameter 3 to be integer, float given ...\\system\\plugins\\webinterface.php on line 680,`

Exactly the same Yellow installation on XAMPP 5.6.8 (with PHP 5.6.8) on Windows 7 doesn't produce this issue. Unfortunately I can't verify whether the same occurs on Linux. Perhaps someone else can do this to find out whether this is Windows specific or PHP 7.x specific.

**Comment by markseuffert** [comment, marked as answer]
Thanks for reporting it. I checked the source code and apparently Yellow sets a cookie too far in the future, which doesn't work on 32-bit versions of PHP. It looks like the cookie time calculation was wrong and caused login problems. Please update your `system/plugins/webinterface.php` and try again.

Should be working now, at least until [2038](https://en.wikipedia.org/wiki/Year_2038_problem). :smile:

**Comment by bpgs** [comment]
Thank you, everything fine now.
